website/sidebars.js

```diff
@@ -55,11 +55,11 @@ module.exports = {
     'API': [
       'api/index',
       'api/core',
-      'api/ui',
-      'api/server',
       'api/graphics',
-      'api/types',
+      'api/server',
```

Comment:
I'd prefer to preserve the original order for the sidebar - they're arranged from most-relevant to least-relevant.

Reply:
suspected that. they looked purposefully arranged. fixing...

Reply:
@lo5, shall I add `ui_ext` under `ui` like this?

```js
    'API': [
      'api/index',
      'api/core',
      'api/ui',
      'api/ui_ext',
      'api/server',
      'api/graphics',
      'api/types',
      'api/test',
    ],
```

Reply:
Yes, sounds good.

Reply:
done!